scripts/case_selection/workflow_case_selection.R

```diff
@@ -76,8 +79,27 @@ prov_exec_selection <- "https://github.com/Sage-Bionetworks/genie-bpc-pipeline/t
 prov_exec_add <- prov_exec_selection
 prov_exec_report <- "https://github.com/Sage-Bionetworks/genie-bpc-pipeline/tree/develop/scripts/case_selection/perform_case_selection.Rmd"
 
+#' Get Main GENIE clinical file using release version name
+#'
+#' @param release Release version name for a GENIE consortium release
+#'                
+#' @return A named list of Main GENIE clinical file Synapse ID.
+get_main_genie_clinical_id <- function(release){
+  query <- glue("SELECT id FROM syn17019650 WHERE name = '{release}'")
+  release_folder_id <- as.character(unlist(as.data.frame(synTableQuery(query, includeRowIdAndRowVersion = F))))
+  release_files <- as.list(synGetChildren(release_folder_id, includeTypes = list("link")))
+  for (release_file in release_files){
+    if (release_file$name == "data_clinical.txt"){
+      return(release_file$id)
+    }
+  }
```

Comment:
I added a check at a earlier step and it will error out if release version is invalid.

Reply:
In the same function?

Reply:
Yup. Like this:
```
get_main_genie_clinical_id <- function(release){
  query <- glue("SELECT id FROM syn17019650 WHERE name = '{release}'")
  release_folder_id <- as.character(unlist(as.data.frame(synTableQuery(query, includeRowIdAndRowVersion = F))))
  if (length(release_folder_id) == 0) {
  stop(glue("The release version is invalid."))
}
  release_files <- as.list(synGetChildren(release_folder_id, includeTypes = list("link")))
  for (release_file in release_files){
    if (release_file$name == "data_clinical.txt"){
      return(release_file$id)
    }
  }
}
```

Reply:
Oh gotcha, I guess I meant having a return statement just for good practice since we're relying on default behavior of R functions where if `if (release_file$name == "data_clinical.txt")` is never true, it will return `NULL`

Reply:
hmmm I thought data_clinical.txt will always be available if we do a consortium release.

Reply:
That's why this is just a `nit`. Yeah, it technically should always be available but with data/code you never know haha

Reply:
Added